HeliumExample/build.gradle

```diff
@@ -22,42 +23,41 @@ android {
         }
     }
 
+    viewBinding {
+        enabled = true
+    }
+
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_11
         targetCompatibility JavaVersion.VERSION_11
     }
-    buildFeatures {
-        viewBinding = true
-    }
-    kotlinOptions {
-        jvmTarget = '1.8'
-    }
 }
 
 dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.1.3'
     implementation 'androidx.appcompat:appcompat:1.4.1'
 
     // Helium
-    implementation 'com.chartboost:helium:2.5.0'
+    implementation 'com.chartboost:helium:2.7.1'
 
     // Helium requirement
-    implementation 'org.greenrobot:eventbus:3.2.0'
+    implementation 'org.greenrobot:eventbus:3.3.1'
 
     // Helium Adapters
-    implementation 'com.chartboost:helium-tapjoy:2.5.0.0'
-    implementation 'com.chartboost:helium-facebook:2.5.0.0'
-    implementation 'com.chartboost:helium-adcolony:2.5.0.0'
-    implementation 'com.chartboost:helium-admob:2.5.0.0'
-    implementation 'com.chartboost:helium-vungle:2.5.0.0'
-    implementation 'com.chartboost:helium-applovin:2.5.0.0'
-    implementation 'com.chartboost:helium-unityads:2.5.0.0'
-    implementation 'com.chartboost:helium-ironsource:2.5.0.0'
-    implementation 'com.chartboost:helium-fyber:2.5.0.0'
-    implementation 'com.chartboost:helium-inmobi:2.5.0.0'
-    implementation 'com.chartboost:helium-mintegral:2.5.0.0'
+    implementation 'com.chartboost:helium-tapjoy:2.7.1.0'
+    implementation 'com.chartboost:helium-facebook:2.7.1.0'
+    implementation 'com.chartboost:helium-adcolony:2.7.1.0'
+    implementation 'com.chartboost:helium-admob:2.7.1.0'
+    implementation 'com.chartboost:helium-vungle:2.7.1.0'
+    implementation 'com.chartboost:helium-applovin:2.7.1.0'
+    implementation 'com.chartboost:helium-unityads:2.7.1.0'
+    implementation 'com.chartboost:helium-ironsource:2.7.1.0'
+    implementation 'com.chartboost:helium-fyber:2.7.1.0'
+    implementation 'com.chartboost:helium-inmobi:2.7.1.0'
+    implementation 'com.chartboost:helium-mintegral:2.7.1.0'
+
     //Tapjoy
-    implementation 'com.tapjoy:tapjoy-android-sdk:12.8.1'
+    implementation 'com.tapjoy:tapjoy-android-sdk:12.9.0'
```

Comment:
are these not automatically pulled in via the above adapters?

Reply:
Yes, but this helps check partner version we currently support for reference.